lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -655,6 +674,30 @@ export abstract class CacheManager implements ICacheManager {
         return key.indexOf(APP_METADATA) !== -1;
     }
 
+    /**
+     * Returns the specific credential (IdToken/AccessToken/RefreshToken) from the cache
+     * @param key
+     * @param credType
+     */
+    private getSpecificCredential(key: string, credType: string): ValidCredentialType {
+        let entity: ValidCredentialType;
+        switch (credType) {
+            case CredentialType.ID_TOKEN: {
+                entity = this.getIdTokenCredential(key);
+                break;
```

Comment:
Nitpick: return immediately
```suggestion
                return this.getIdTokenCredential(key);
```

Reply:
Addressed this.